src/CONST.ts

```diff
@@ -1084,6 +1084,7 @@ const CONST = {
     SAVE_WITH_EXPENSIFY_URL: `${USE_EXPENSIFY_URL}/savings-calculator`,
     CFPB_PREPAID_URL: 'https://cfpb.gov/prepaid',
     STAGING_NEW_EXPENSIFY_URL: 'https://staging.new.expensify.com',
+    TESTING_NEW_EXPENSIFY_URL: `https://${Config?.PULL_REQUEST_NUMBER}.pr-testing.expensify.com`,
```

Comment:
```suggestion
    PR_TESTING_NEW_EXPENSIFY_URL: `https://${Config?.PULL_REQUEST_NUMBER}.pr-testing.expensify.com`,
```

Should we have a fallback if `Config?.PULL_REQUEST_NUMBER` is `undefined`?

Reply:
Could you apply these changes @pac-guerreiro ? (`PR_TESTING_NEW_EXPENSIFY_URL`)